src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -486,6 +489,133 @@ trait BronzeTransforms extends SparkSessionWrapper {
     }
   }
 
+  def prepClusterSnapshot(
+                           workspace: Workspace,
+                           pipelineSnapTS: TimeTypes,
+                           apiEnv: ApiEnv,
+                           database: Database,
+                           clusterSnapshotErrorsTarget: PipelineTable,
+                           config: Config,
+                           apiEndpointTempDir: String
+                         )(auditDF: DataFrame) : DataFrame = {
+
+    val clusterIDs = auditDF
+      .select(cluster_idFromAudit.alias("cluster_id"))
+      .filter(col("cluster_id").isNotNull)
+      .distinct()
+      .select("cluster_id").distinct().collect().map(x => x(0).toString)
+
+    var jobClusterIDs = Array[String]()
+    try {
+      val jobDF = getJobsBase(auditDF)
+      if (!jobDF.isEmpty) {
+        jobClusterIDs = jobDF.select('clusterId.alias("cluster_id")).distinct().collect().map(x => x(0).toString)
+      }
+    } catch {
+      case e: Exception => {
+        val message = s"No New records are their for Job Cluster in Audit Log: ${e.getMessage}"
```

Comment:
Spelling on this message. It should be: "There are no new job clusters in Audit log"

Reply:
Done